test/audit/Chambers.t.sol

```diff
@@ -0,0 +1,192 @@
+import {StreamingFeeWizard, IStreamingFeeWizard} from "src/StreamingFeeWizard.sol";
+import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import {console} from "forge-std/console.sol";
+
+contract FakeChamber {
+    function mint(address, uint256) external {
+        return;
+    }
+
+    function decimals() external view returns (uint8) {
+        return 18;
+    }
+
+    function getConstituentsAddresses() external view returns (address[] memory) {
+        address[] memory constituents = new address[](1);
+
+        // USDC
+        constituents[0] = 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+
+        return constituents;
+    }
+
+    function getConstituentQuantity(address) external view returns (uint256) {
+        return 1e18;
+    }
+}
+
+contract ChambersTest is Test {
+    uint256 public constant FORK_BLOCK_NUMBER = 16_421_844;
+
+    ERC20 public constant USDC = ERC20(0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48);
+
+    ChamberGod public god;
+    IssuerWizard public issuer;
+    RebalanceWizard public rebalancer;
+    StreamingFeeWizard public fees;
+    Chamber public chamber;
+
+    address public owner;
+    address public manager;
+    address public bob;
+    address public alice;
+
+    function setUp() public {
+        //vm.createSelectFork(vm.rpcUrl("mainnet"), FORK_BLOCK_NUMBER);
+
+        owner = makeAddr("OWNER");
+        manager = makeAddr("MANAGER");
+        bob = makeAddr("BOB");
+        alice = makeAddr("ALICE");
+
+        vm.startPrank(owner);
+
+        //[ARCH] Moved god a few lines above because address is needed at IssuerWizard constructor
+        god = new ChamberGod();
+        issuer = new IssuerWizard(address(god));
+        rebalancer = new RebalanceWizard();
+        fees = new StreamingFeeWizard();
+
+        god.addWizard(address(issuer));
+        god.addWizard(address(rebalancer));
+        god.addWizard(address(fees));
+
+        address[] memory constituents = new address[](1);
+        constituents[0] = address(USDC);
+
+        uint256[] memory quantities = new uint256[](1);
+        quantities[0] = 1000e6;
+
+        address[] memory wizards = new address[](3);
+        wizards[0] = address(issuer);
+        wizards[1] = address(rebalancer);
+        wizards[2] = address(fees);
+
+        address[] memory managers = new address[](1);
+        managers[0] = manager;
+
+        chamber = Chamber(
+            god.createChamber("Nomoi USDC", "nUSDC", constituents, quantities, wizards, managers)
+        );
+
+        vm.stopPrank();
+
+        vm.startPrank(manager);
+        IStreamingFeeWizard.FeeState memory feeState =
+            IStreamingFeeWizard.FeeState(owner, 1e18, 1e18, block.timestamp);
+
+        fees.enableChamber(chamber, feeState);
+        vm.stopPrank();
+
+        deal(address(USDC), bob, 1000000e6);
+        deal(address(USDC), alice, 1000000e6);
+    }
+
+    function testCollectMultiple() public {
```

Comment:
Missing test description, and if its whether SUCCESS or REJECT. Add global sections as well

Reply:
will delete it since is not relevant for the fixes.